packages/contracts-bedrock/src/dispute/FaultDisputeGameN.sol

```diff
@@ -1154,14 +1155,15 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
             : _firstValidRightIndex(_pos.traceAncestorBounded(SPLIT_DEPTH), N_BITS);
 
         uint256 offset = ancestorPos_.raw() % (1 << N_BITS);
-        if (MAX_ATTACK_BRANCH == offset) {
+        // If ancestorPos_.raw() == 1 (rootclaim), we don't offset it
+        if (MAX_ATTACK_BRANCH == offset || ancestorPos_.raw() == 1) {
```

Comment:
Let offset be 0 when its rootClaim, so the meaning of `traceAncestorPosValue ` will be consistent.